src/Akka.Hosting/AkkaHostingExtensions.cs

```diff
@@ -58,6 +59,12 @@ public static IServiceCollection AddAkka(this IServiceCollection services, strin
         public static IServiceCollection AddAkka<T>(this IServiceCollection services, string actorSystemName, Action<AkkaConfigurationBuilder, IServiceProvider> builder) where T:AkkaHostedService
         {
             var b = new AkkaConfigurationBuilder(services, actorSystemName);
+            
+            // add the default Akka.Streams configuration by default - hurts nothing, but
+            // ensures that StreamRefs work correctly out of the box in case users
+            // haven't attempted to materialize a stream yet
+            b.AddHocon(ActorMaterializer.DefaultConfig(), HoconAddMode.Append);
```

Comment:
Added the HOCON here - it can be in Append mode because it's just the defaults - it can go at the end of the line.